astroplan/plots/time_dependent.py

```diff
@@ -145,40 +147,49 @@ def plot_airmass(target, observer, time, ax=None, style_kwargs=None,
                       'to use a scalar. (Or maybe a list with length > 1?).',
                       PlotWarning)
 
-    # Calculate airmass
-    airmass = observer.altaz(time, target).secz
-    # Mask out nonsense airmasses
-    masked_airmass = np.ma.array(airmass, mask=airmass < 1)
+    if not isinstance(targets, list):
```

Comment:
The practical approach would be to use `Iterable` because that's the prerequisite of the `for` loop (you need an iterable to do a for loop). But I think the only (rare) real use-case where it makes a difference is a generator, which is an iterable but not a sequence.